src/clarity/ui/src/styles.css

```
@@ -0,0 +1,5 @@
+@import 'tailwindcss';
+@source "../../../utils/ui";
+@import '@ui/theme/styles.css';
```

Stylelint (17.4.0)

[error] Unexpected invalid position `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)